ONSdigital/census-rm-performance-tests
# Clear down queues

Motivation 
https://trello.com/c/zEeGsJMP/1368-empty-rabbitmq-queues-before-test-3
Make sure rabbitMq is empty to give more consistent performance for the performance tests.  Consumable queues with msgs will occupy the apps, end queues like case.rh.case will fill up overtime and degrade rabbitmq performance as it will require paging etc.

What's changed
Added in api calls to get all the queues on the / vhost, then loops over them calling the delete API on them.  It also keeps trying to delete everything on the delayedRedeliveryQueue every loop,  this is to try and stop bad messages getting back on queues after they've been purged.

How to Test
Performance tests should work as normal locally.  I also published a lot of bad 'dfsfs' msgs to a number of consumed queues to exercise the delayedRedeliveryQueue and used the 1000 per treatment code file to create a high number of inflight valid messages, then killed the 1000 test after load was complete.   Then ran tests again and watched in rabbitUI to see if the queues were emptied.
